stylesheets/components/list.css

```diff
@@ -0,0 +1,36 @@
+@import url("/stylesheets/main.css");
+
+.list-item{
+    list-style-type: none;
+    border: 1.5px solid var(--info);
+    color: var(--info);
+    background-color: var(--bg-white);
+    border-radius: 0.1rem;
+    padding: 0.5rem;
+    width: fit-content;
+}
+
+.list{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+    margin: 0.5rem;
```

Comment:
You can create these properties as individual utility classes and apply it to the list item in the upcoming versions of it!!

Reply:
Will implement it